db/seeds.rb

```diff
@@ -5,3 +5,29 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+ 
+require 'faker'
+
+user = User.create!({
+  email: "test@email.come",
```

Comment:
Typo - Can you remove the 'e' from the end of email string?

Reply:
Ok Thanks